e2e-tests/src/tests/studies/ourhealth/mailing-list.test.ts

```diff
@@ -1,9 +1,8 @@
 import { test, expect } from '@playwright/test'
-import { adminLogin, getParticipantUrl, randomChars } from 'tests/e2e-utils'
+import { adminLogin, randomChars } from 'tests/e2e-utils'
 
 test('shows mailing list dialog on direct link', async ({ page }) => {
-  const participantUrl = getParticipantUrl('demo')
-  await page.goto(`${participantUrl}/?showJoinMailingList=true `)
+  await page.goto(`${process.env.PARTICIPANT_URL}/?showJoinMailingList=true `)
```

Comment:
the `PARTICIPANT_URL` env variable takes into account whether or not the test is running in CI. `getParticipantUrl` was hardcoded to assume that this was running locally